rama-http/src/layer/har/mod.rs

```diff
@@ -0,0 +1,23 @@
+use crate::layer::har::spec::Log as HarLog;
+use std::future::Future;
+
+pub mod default;
+pub mod layer;
+pub mod service;
+pub mod spec;
+
+#[derive(Clone)]
+pub struct Comment {
+    pub author: String,
+    pub text: String,
+}
+
+pub trait Toggle {
+    fn toggle(&mut self) -> impl Future<Output = bool> + Send + '_;
+    fn is_recording_on(&self) -> bool;
+}
+
+pub trait Recorder: Clone + Send + Sync + 'static {
+    fn record(&self, line: HarLog) -> impl Future<Output = ()> + Send + '_;
+    fn data(&self) -> Vec<HarLog>;
```

Comment:
this should not be needed, as the data is stored by the recorder... How it is stored is up to that. The recorder acts like a "sink".

Reply:
I fully understand - I removed the data() method, also from the default (which is not being used anyway)